src/editor.rs

```diff
@@ -267,52 +267,32 @@ fn parse_command(keymap: &KeyMap, command: &str) -> Option<Action> {
     let mut command_char_iter = command.chars();
 
     // Consume the first char of the command
-    if let Some(c) = command_char_iter.next() {
```

Comment:
Could you add `#[rustfmt_skip]` to the function so that I don't reformat it when I save the file (I have format-on-save enabled to avoid loads of mess with commits that are only formatting).

Reply:
Apparently you actually need `#[cfg_attr(rustfmt, rustfmt_skip)]` to make this work on stable.

Reply:
This is how `rustfmt` formatted it, so it shouldn't change it anymore. Or am I missing something?

Reply:
No you're quite right - this is how rustfmt formats it. I was very tired last night and completely forgot that the PR CI checks also run rustfmt and so this _had_ to be the way rusrfmt formatted it :laugh:.